expect.yaml

```diff
@@ -0,0 +1,67 @@
+# Partly generated from https://git.alpinelinux.org/aports/plain/main/expect/APKBUILD
+package:
+  name: expect
+  version: 5.45.4
+  epoch: 0
+  description: A tool for automating interactive applications
+  copyright:
+    - license: NIST-PD
+  dependencies:
+    runtime:
+      - tcl
+      - libstdc++
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libstdc++
+      - tcl-dev
+      - wget
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 49a7da83b0bdd9f46d04a04deec19c7767bb9a323e40c4781f89caf760b92c34
+      uri: https://downloads.sourceforge.net/project/expect/Expect/${{package.version}}/expect${{package.version}}.tar.gz
+
+  - runs: |
+      wget -T5 --dns-timeout=20 --tries=5 --random-wait --retry-on-http-error=429 --retry-connrefused -r -np -A "*.patch" -nH -e robots=off --cut-dirs=4 https://git.alpinelinux.org/aports/plain/main/expect/
```

Comment:
can't we just put those patches here? you can create a `expect` folder and put patches there.

Reply:
I think it's possible. But somewhat I can't test the build process from my local env.
Given below steps to build `zetasql` (package with patches that I can find) in `os` folder.
1. generate key
```bash
docker run --privileged --rm -v "${PWD}":/work  --workdir /work cgr.dev/chainguard/melange  keygen local-melange.rsa
```
2. build (copied from [Action log](https://github.com/wolfi-dev/os/actions/runs/9784294846/job/27014905902?pr=23127)
```bash
docker run --privileged --rm -v "${PWD}":/work  --workdir /work cgr.dev/chainguard/melange  build zetasql.yaml --repository-append ./packages --keyring-append local-melange.rsa.pub --signing-key local-melange.rsa --arch x86_64 --env-file build-x86_64.env --namespace wolfi --generate-index false  --pipeline-dir ./pipelines/ --create-build-log --cache-dir=.melangecache -k https://packages.wolfi.dev/os/wolfi-signing.rsa.pub -r https://packages.wolfi.dev/os
```
In local env, it always gives:
```text
2024/07/12 02:11:20 WARN /bin/sh: can't open multi-arch-for-linux.patch: no such file
```

I'm testing but can't tell there is anything wrong in my setup. So right now I'm using wget to fetch resource.

Reply:
OK, I found the issue,
Due to info mismatch in [Makefile](https://github.com/wolfi-dev/os/blob/2211ce27a071b9f144606372e580195585402bb2/Makefile#L110) that, currently not reflect current state of actual command.
https://github.com/wolfi-dev/os/blob/2211ce27a071b9f144606372e580195585402bb2/Makefile#L110-L111

So, I can test the build now, gonna push asap.